docs/core/diagnostics/dotnet-trace.md

```diff
@@ -236,13 +236,23 @@ dotnet-trace convert [<input-filename>] [--format <Chromium|NetTrace|Speedscope>
 ## dotnet-trace ps
 
  Lists the dotnet processes that traces can be collected from.
+ On .NET 7 and later versions, also displays the command-line arguments that each process was started with.
 
 ### Synopsis
 
 ```console
 dotnet-trace ps [-h|--help]
 ```
 
+### Example
+
```

Comment:
What do you think about expanding to something like this?

### Example

Suppose you start a long-running app using the command `dotnet run --configuration Release`. In another window, you run the `dotnet-trace ps` command. The output you'll see is as follows. The command-line arguments, if any, are shown in .NET 7 and later versions.

```console
> dotnet-trace ps
  
     21932 dotnet     C:\Program Files\dotnet\dotnet.exe     run --configuration Release
     36656 dotnet     C:\Program Files\dotnet\dotnet.exe
```